fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanSchedularServiceImpl.java

```diff
@@ -355,4 +396,26 @@ private AppUser getAppUserIfPresent() {
         return user;
     }
 
+    private void buildNotification(String permission, String objectType, Long objectIdentifier, String notificationContent,
+            String eventType, Long appUserId, Long officeId) {
+
+        String tenantIdentifier = ThreadLocalContextUtil.getTenant().getTenantIdentifier();
+        Set<Long> userIds = getNotifiableUserIds(officeId, permission);
+        NotificationData notificationData = new NotificationData(objectType, objectIdentifier, eventType, appUserId, notificationContent,
+                false, false, tenantIdentifier, officeId, userIds);
+        try {
+            notificationEventPublisher.broadcastNotificationLoanRepaymentReminders(notificationData,
+                    this.env.getProperty("LOAN_REPAYMENT_REMINDER_QUEUE"));
```

Comment:
fetch property with 
 this.env.getProperty("fineract.activemq.loanRepaymentReminderQueue");?

Reply:
fixed